tests/coroutines/tgc.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   matrix: "--gc:refc; --gc:arc; --gc:orc"
+  disabled: true
```

Comment:
Could you add a readme for this directory stating that we disabled it, and the replacement for it will serve similar purposes but will be CPS based.

Reply:
Make sense, I will remove the directory.

Reply:
done